### ISSUE TITLE
Rails won't start after adding neo4j gem

### ISSUE THREAD

**Issue by RauerA** [neo4jrb/neo4j#1039]
I've added the neo4j gem to my gem file and than run bundle install. All dependencies were installed correct.

After I added (second line only)
require 'rails/all'
require 'neo4j/railtie'

to application.rb the rails console and the rails server won't start any more. No error message, no output. Just a empty command line.

Neo4j gem version: 5.0.0
Rails version: 4.2.1
Ruby version: 2.2.1p85
Neo4j version: 2.3.0 unix (community)

**Comment by subvertallchris** [comment]
Start by updating to v5.2.14 since 5.0.0 isn’t supported with Neo4j 2.3.0. I’m not aware of this issue ever being reported and there’s nothing specific in the newer version that I think will correct this but it’s a healthy place to start. Otherwise, check your application and system log files. What OS?

Chris

On November 10, 2015 at 11:04:30 AM, Andre Rauer (notifications@github.com) wrote:

I've added the neo4j gem to my gem file and than run bundle install. All dependencies were installed correct.

After I added (second line only)
require 'rails/all'
require 'neo4j/railtie'

to application.rb the rails console and the rails server won't start any more. No error message, no output. Just a empty command line.

Neo4j gem version: 5.0.0
Rails version: 4.2.1
Ruby version: 2.2.1p85
Neo4j version: 2.3.0 unix (community)
